bin/install-wp-tests.sh

```diff
@@ -196,14 +196,14 @@ install_wc() {
   if [ ! -f "$WC_VERSION_FILE" ]; then
     # set up testing suite
     rm -rf "$WC_DIR"
-    mkdir -p "$WC_DIR"
     echo "Installing WooCommerce ($WC_VERSION)."
     # Grab the necessary plugins.
     WC_TMPDIR="${TMPDIR}/woocommerce-${WC_VERSION}"
     rm -rf "${WC_TMPDIR}"
     git clone --quiet --depth=1 --branch="${WC_VERSION}" https://github.com/woocommerce/woocommerce.git "${WC_TMPDIR}"
-    mv "${WC_TMPDIR}"/plugins/woocommerce/* "$WC_DIR"
-	touch "$WC_VERSION_FILE"
+
+    ln -s "${WC_TMPDIR}"/plugins/woocommerce "$WC_DIR"
+	  touch "$WC_VERSION_FILE"
```

Comment:
Just a small note here about mixed spacing. This line uses a tab and 2 spaces.

Reply:
Sharp-eyed! Fixed here [2583950](https://github.com/woocommerce/google-listings-and-ads/pull/2625/commits/2583950f750d5f03e270290bab919c04f9faf542)